GeneralsMD/Code/GameEngine/Source/GameNetwork/GameSpy/Thread/BuddyThread.cpp

```diff
@@ -308,7 +308,8 @@ void BuddyThreadClass::Thread_Function()
 				break;
 			case BuddyRequest::BUDDYREQUEST_DELETEACCT:
 				m_isdeleting =  true;
-				gpDeleteProfile( con );
+				// API was updated since Generals released to require a callback. Passing -1 will make our wrapper ignore this.
```

Comment:
Comments we add to code need to be identifiable to distinguish between EA and community.

Suggested comment style is

`// TheSuperHackers @tweak OmniBlade 04/03/2025 API was updated since Generals released to require a callback. Passing -1 will make our wrapper ignore this.`

Author and date can be omitted.

Reply:
I've force pushed the PR and fixed these comment suggestions.